gl/src/lib.rs

```diff
@@ -27,12 +27,20 @@ use std::time::Duration;
 
 pub struct GLDevice {
     version: GLVersion,
+    default_framebuffer_id: GLuint,
 }
 
 impl GLDevice {
     #[inline]
     pub fn new(version: GLVersion) -> GLDevice {
-        GLDevice { version }
+        let mut default_framebuffer_id = 0;
+        unsafe {
+            gl::GetIntegerv(gl::FRAMEBUFFER_BINDING, &mut default_framebuffer_id);
```

Comment:
This can be slow, and it's hidden global state. I think it's probably better to just pass in the framebuffer as a parameter.